packages/core/README.md

```diff
@@ -3,11 +3,53 @@
 LoopBack makes it easy to build modern applications that require complex
 integrations.
 
+# Overview
+
 - Fast, small, powerful, extensible core
 - Generate real APIs with a single command
 - Define your data and endpoints with OpenAPI
 - No maintenance of generated code
 
-# License
+
+## Installation
+
+```shell
+$ npm install --save-dev @loopback/core
+```
+
+_This package is typically used in tests, save it to `devDependencies` via `--save-dev`._
+
+## Basic use
+
+  ```ts
+  const app = new Application();
+
+  //Register a controller class with this application.
+
+   class MyController {
+     greet(name: string) {
+       return `hello ${name}`;
+     }
+   }
+   app.route('get', '/greet', operationSpec, MyController, 'greet');
```

Comment:
Let's use the example from https://github.com/strongloop/loopback-next/wiki/Hello-World:

```ts
import {Application} from '@loopback/core';

const app = new Application();
app.handler((sequence, request, response) => {
  sequence.send(response, 'hello world'));
});

(async function start() {
  await app.start();
  console.log(`The app is running on port ${app.getSync('http.port')}`);
})();
```